src/main/steps/c100-rebuild/safety-concerns/abduction/template.njk

```diff
@@ -1,7 +1,7 @@
 <p class="govuk-body-m">
-<a href="{{guidanceOnAbductionLink}}" id="guidanceOnAbduction" target="_blank" class="govuk-link">{{guidanceOnAbductionLabel}}</a>
-</br>
-<a href="{{getHelpOnReturningChildAbroadLink}}" id="getHelpOnReturningChildAbroad" target="_blank" class="govuk-link">{{getHelpOnReturningChildAbroadLabel}}</a>
-</br>
-<a href="{{preventChildFromObtainingPassportLink}}" id="preventChildFromObtainingPassport" target="_blank" class="govuk-link">{{preventChildFromObtainingPassportLabel}}</a>
-</p>
\ No newline at end of file
+<a href="{{guidanceParentalChildHyperlink}}" id="guidanceParentalChildLabel" target="_blank" class="govuk-link">{{ guidanceParentalChildLabel }}</a>
+<br>
+<a href="{{helpForAbroadChildHyperlink}}" id="helpForAbroadChildLabel" target="_blank" class="govuk-link">{{ helpForAbroadChildLabel }}</a>
+<br>
+<a href="{{stopChildGettingPassportHyperlink}}" id="idestopChildGettingPassportLabelntifySignsOfDomesticAbuseLabel" target="_blank" class="govuk-link">{{ stopChildGettingPassportLabel }}</a>
```

Comment:
@kumarmadurai id field got some junk value here by mistake I think. Can we please correct ?

Reply:
Fixed